### PR TITLE
Admin updates!

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This pull request adds a couple of updates to administering OAuth clients:

:gear: Adds the `allowed_grants` field to index, show, and edit pages.

🔗 Adds support for adding Redirect URLs as a comma-separated list.

📄 Updates the setup documentation to be in line with our other apps.

### CODE REVIEW

**Comment by DFurnes** [comment]
Gonna ignore the style complaints again for this PR, will make a follow-up to clean up.

File: .env.example [hunk 1]
```diff
@@ -11,7 +11,7 @@ CACHE_DRIVER=file
 SESSION_DRIVER=file
 QUEUE_DRIVER=sync
 
-NORTHSTAR_URL=http://northstar.dev:8000
+NORTHSTAR_URL=http://northstar.dev
```
Comment (weerd) [inline .env.example]: 🎉  I hate the port specification!

Comment (DFurnes) [reply .env.example]: It's outta here! 🔥

File: app/Http/Controllers/ClientController.php [hunk 2]
```diff
@@ -75,7 +75,14 @@ public function create()
      */
     public function store(Request $request)
     {
-        $this->northstar->createNewClient($request->all());
+        $parameters = $request->all();
+
+        // If more than one `redirect_uri` is provided, parse into an array.
+        if (str_contains($parameters['redirect_uri'], ',')) {
+            $parameters['redirect_uri'] = array_map('trim', explode(',', $parameters['redirect_uri']));
```
Comment (weerd) [inline app/Http/Controllers/ClientController.php]: noice ;)

File: app/Http/Controllers/ClientController.php [hunk 3]
```diff
@@ -75,7 +75,12 @@ public function create()
      */
     public function store(Request $request)
     {
-        $this->northstar->createNewClient($request->all());
+        $parameters = $request->all();
+
+        // Transform 'redirect_uri' from a CSV into an array of strings.
+        $parameters['redirect_uri'] = array_map('trim', explode(',', $parameters['redirect_uri']));
```
Comment (weerd) [inline app/Http/Controllers/ClientController.php]: Could this be a helper function so as not to repeat below?

Comment (DFurnes) [reply app/Http/Controllers/ClientController.php]: Mmm, that's a pretty good idea.

**Pull request merged**